src/index.html

```diff
@@ -17,6 +17,93 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
```

Comment:
Class names should represent the meaning of the content. The class `nav_list` should be `nav__list` to follow the BEM (Block Element Modifier) naming convention.

Reply:
+